WarOfEmpires.QueryHandlers/Empires/GetTroopsQueryHandler.cs

```diff
@@ -40,8 +40,7 @@ public sealed class GetTroopsQueryHandler : IQueryHandler<GetTroopsQuery, TroopM
                 HasUpkeepRunOut = player.HasUpkeepRunOut,
                 CurrentStamina = player.Stamina,
                 HasSoldierShortage = player.GetSoldierRecruitsPenalty() > 0,
-                StaminaToFull = 100 - player.Stamina,
-                //TODO: Implement this with better name HealMaxAfford = ;
+                StaminaToHeal = (player.GetStaminaToHeal(100 - player.Stamina)).ToString()
```

Comment:
See other comment about removing the parameter; you can move the calculation to the function itself.

Reply:
Parameter removed